resources/gin/train/tracknet_v2_1_classifier_train.cfg

```diff
@@ -1,10 +1,10 @@
 ### experiment setup ###
-experiment.model = @Classifier
+experiment.model = @ariadne.tracknet_v2_1.model.Classifier
```

Comment:
You may use such notation `@Classifier` if there is not another class called `Classifer`. Also I suggest to rename the class to something similar to `TracknetClassifier`

Reply:
changed to TrackNetClassifier